src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,4 @@
+$color-main: #060B35;
+$color-secondary: #616070;
+$color-blue: #00ACDC;
+$star: 4;
```

Comment:
what is `star`? its better to give more informative names according to the value